addons/udes_stock/tests/test_change_quant_location_wizard.py

```diff
@@ -0,0 +1,108 @@
+from . import common
+from odoo.exceptions import ValidationError
+
+
+class TestChangeQuantLocationWizard(common.BaseUDES):
+    @classmethod
+    def setUpClass(cls):
+        """Setup basic data"""
+        super(TestChangeQuantLocationWizard, cls).setUpClass()
+        Picking = cls.env["stock.picking"]
+        cls.change_quant_location = cls.env["change_quant_location"].sudo(cls.outbound_user)
+
+        cls.package1 = cls.create_package()
+        cls.quant1 = cls.create_quant(
+            cls.apple.id,
+            cls.test_location_01.id,
+            10,
```

Comment:
(Feel free to ignore/resolve) Nit: Quantity of 10 is set here for `quant_1` and also for `quant_2` in `test_create_picking_multiple_packages`, is it worth adding in a class variable? e.g. `cls.product_quantity = 10`